_posts/2016-05-12-Developers-love-spaces.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Developers love spaces - Solita Developer Survey 2016 results
+author: arto
+excerpt: Now we know what developers love and love not this year
+categories: 
+- Java
+tags: 
+- Java
+- Clojure 
+- Javascript
+- Development
+- Culture
+- Solita
+---
+
+## What do devs love?
+
+Well a lot of things, apparently. Today, being a dev is about being passionate about things. But in this case, I think I already let out a spoiler in the title. Devs do love spaces. And, thanks to recent results from Solita Developer Survey, I got the stats to prove it. According the the survey, crushing percentage of 72,2% of all solita devs indent their code with spaces, as opposed to tabs. 2% didn't understand the question, so I suppose they don't indent.
+
+![Would you indent code with tabs or spaces?](/img/devs-love-spaces/spaces-vs-tabs.png)
+
+Other trends are also visible in the survey: for maintaining configurations - an important part of doing devops, or doing things right - there are many tool choices available. Here in Solita, the preferred choice is Ansible - with all its pros and cons 42% of devs choose it for automating and documenting the configurations. Chef and Puppet together don't come even close. Many still use traditional shell scripting (21%), or have to do manual work (16%). Another way to look at these statistics is to say that 84% of developers work in happily automated configuration management world, while the rest 16% just have to suck it.
```

Comment:
documenting the configurations -> voisi olla sidosteisempi "their configurations"
in *a* happily automated...